package-lock.json

```diff
@@ -1904,8 +1904,7 @@
     },
```

Comment:
@EnMod are these changes to top-level package-lock expected?

Reply:
@alexcarpenter Ah, these snuck through.  Probably worth reverting the changes removing `"resolved"` and `"integrity"` keys

Reply:
Ok changes have been reverted as of these latest commits:
https://github.com/hashicorp/react-components/pull/783/commits/17f413d5f9f682a430fc8e6c3fdf5240e3f22f79
https://github.com/hashicorp/react-components/pull/783/commits/988a47b6cb834e3b300621578455c3ca885136af